### ISSUE TITLE
torch to float?

### ISSUE THREAD

**Issue by zyh722** [XMU-Kuangnan-Fang-Team/GENetLib#141]
I noticed that the final output of your newly updated `prediction` function is in the form of `torch`. Would it be better to change it to the `float` form in `numpy`?

**Comment by Barry57** [comment]
Thanks for your comment. You can use `.detach().numpy()` to convert torch into numpy. Since our neural network architecture is based on torvh type, we suggest not to revise codes in `predict_ge` function.

**Comment by zyh722** [comment]
I see. So how to calculate c_index? Any examples?

**Comment by Barry57** [comment]
Yes, here is an example about `func_ge`:
```
from GENetLib.sim_data import sim_data_func
from GENetLib.func_ge import func_ge
from GENetLib.predict_ge import predict_func
from GENetLib.survival_costfunc_cindex import c_index
import torch
ytype = 'Survival'
num_hidden_layers = 2
nodes_hidden_layer = [100,10]
learning_rate2 = 0.035
Lambda = 0.01
learning_rate1 = 0.02
lambda2 = 0.01
num_epochs = 50
nbasis1 = 5
params1 = 4
func_continuous = sim_data_func(n = 1000, m = 30, ytype = ytype, seed = 123)
#func_continuous = sim_data_func(n = 1000, m = 30, ytype = ytype, input_type = 'func', seed = 123)
y = func_continuous['y']
z = func_continuous['Z']
location = func_continuous['location']
X = func_continuous['X']
func_ge_res = func_ge(y, X, location, z, ytype, 'Bspline', 
                      num_hidden_layers, nodes_hidden_layer, num_epochs, learning_rate1, learning_rate2, 
                      nbasis1, params1, lambda1 = 1.1*lambda2, lambda2 = lambda2, Lambda = Lambda,
                      Bsplines = 5, norder1 = 4, model = None, split_type = 1, ratio = [3, 1, 1],
                      plot_res = True, plot_beta = True)
pred = predict_func(func_ge_res, y, ytype, X, z, location, Bsplines = 5)
cindex = c_index(pred, torch.from_numpy(y[:,0]), torch.from_numpy(y[:,1]))
```

**Comment by Barry57** [comment]
Here, remember to convert numpy in survival output `y` into torch type.
